addons/event_booth_sale/i18n/event_booth_sale.pot

```diff
@@ -326,6 +326,16 @@ msgstr ""
 msgid "Select an event booth"
 msgstr ""
 
+#. module: event_booth_sale
+#. odoo-python
+#: code:addons/event_booth_sale/models/product_template.py:0
+msgid ""
+"The \"service_tracking\" for the product template, %(product_template_name)s"
+" cannot be changed because one of its variants is assigned to the Event "
+"Booth Category, %(event_booth_category_name)s. The service_tracking must "
+"remain \"Event Booth\"."
+msgstr ""
+
```

Comment:
the translation was only necessary for stable version, there is no need for master :slightly_smiling_face: